Sources/BuildSystemIntegration/ExternalBuildSystemAdapter.swift

```diff
@@ -178,6 +185,48 @@ actor ExternalBuildSystemAdapter {
     ).connection
   }
 
+  private func getConfigPath() -> AbsolutePath? {
```

Comment:
Now you’re no longer for the folders in `projectRoot(for workspaceFolder:options:)`. You need to check in both.

Reply:
Let me know if the updated logic is fine, I will then proceed with the tests.

Reply:
Yes, this path looks fine now. I would encourage you to write tests alongside the implementation because you would have easily found this issue with a test case.

Reply:
Indeed.